app/models/user.rb

```diff
@@ -395,6 +395,19 @@ def max_friends_per_hour
     max_friends.clamp(0, Settings.max_friends_per_hour)
   end
 
+  def max_changeset_comments_per_hour
+    if moderator?
+      36000
```

Comment:
Maybe better as "60 * 60 * 10" since I thought that was one per second when I read it at first.